src/plugins/discover/public/context_awareness/profile_providers/register_profile_providers.ts

```diff
@@ -88,7 +89,7 @@ const extractProfileIds = (providers: Array<BaseProfileProvider<{}>>) =>
   providers.map(({ profileId }) => profileId);
 
 const createRootProfileProviders = (_providerServices: ProfileProviderServices) =>
-  [] as RootProfileProvider[];
+  [createSecurityRootProfileProvider(_providerServices)] as RootProfileProvider[];
```

Comment:
@davismcphee , do you think it make sense, if I add `isEnabled` flag to profiles. 

This way, experimental profiles will always have `isEnabled` as `No` and it can be overidden by `discover.experimental.enabledProfiles`.

Reply:
Nice, I like this approach better! My only suggestion is rather than using an `isEnabled` flag that every profile needs to add, what if we go with something like an optional `isExperimental` flag to explicitly mark only the experimental profiles?

Reply:
Works for me. Done changes here: 0739ef2f137.